### PR TITLE
CHROMEOS cros-tast.jinja2: Add more tast tests (tabswitch + videodecode)

### PR DESCRIPTION
**Pull request by nuclearcat**
It's time to expand the number of tast tests and include more interesting ones.
One of the tests switches different tabs in chrome, the second tests video decoding.

Signed-off-by: Denys Fedoryshchenko <denys.f@collabora.com>

### CODE REVIEW

**Comment by nuclearcat** [comment]
As test shows: https://storage.chromeos.kernelci.org/kernelci/chromeos-stable/chromeos-stable-20220712.0/x86_64/cros---chromeos-5.10-x86_64-chromeos-intel-pineview.flavour.config+x86-chromebook/clang-14/lab-collabora/cros-tast-fixed-hp-x360-12b-ca0010nr-n4020-octopus_chromeos.html
we need higher timeout, as tast tests doesnt fit in 5 minutes. I will set it to 15 minutes.

**Comment by nuclearcat** [comment]
We are getting results! Video decoding works
https://chromeos.kernelci.org/test/plan/id/62cf8317763c686e277df429/
By some reasons tabswitch test fails, which i need to investigate, probably there is more appropriate test

**Comment by nuclearcat** [comment]
Updated tests to TabSwitchCUJ2 and added WindowCyclePerf.
TabSwitchCUJ seems fail too often and doesnt produce proper results.

**Comment by gctucker** [comment]
That's great news. Also it seems the video ones don't take very long to run.  I was thinking we should start making separate test plans with different lists of Tast tests but for now we can just bundle all these into `cros-tast` and see how it goes.

**Comment by nuclearcat** [comment]
TabSwitchCUJ2 require additional parameters (which i didnt found documentation yet, so i will leave it for later). Replaced by another tast tests which is proven to fail on QEMU.

**Comment by nuclearcat** [comment]
https://lava.collabora.dev/scheduler/job/6836701
ui.WindowCyclePerf new test worked fine on hatch
 ui.TabSwitchCUJRecorder2.local depending on external URLs and a bit long, probably it should be separate tast test?

**Comment by gctucker** [comment]
The job took 11min, I think it's fine to leave it as a single `cros-tast` for now. Up to 15min shouldn't cause congestion in the lab. Once we've experimented with a few Tast tests we can then start creating new test plans by grouping them.

File: config/lava/chromeos/cros-tast.jinja2 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 - test:
     namespace: chromeos
     timeout:
-      minutes: 5
+      minutes: 15
```
Comment (gctucker) [inline config/lava/chromeos/cros-tast.jinja2:7]: The job took 11min to run, I think the timeout should be set to something like 30min. In fact this should be a template variable like for other test plans.

Comment (nuclearcat) [reply config/lava/chromeos/cros-tast.jinja2]: Done, updated

Comment (gctucker) [reply config/lava/chromeos/cros-tast.jinja2]: Thanks. Then maybe as a follow-up this could be turned into a template parameter with a new PR.

**Comment by nuclearcat** [comment]
https://lava.collabora.dev/scheduler/job/6883647
Test passing successfully on 5.10 upstream kernel

**Comment by mgalka** [comment]
I'm not entirely sure if this sentence should be a part of the commit message:

> It's time to expand the number of tast tests and include more interesting ones.

I'd rather focus on the information on what tests have been added and what is their purpose.

**Comment by nuclearcat** [comment]
Done, updated

**Review by mgalka — APPROVED** [review]

**Pull request merged**